<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Avogadro::QtPlugins::CondaDialog</class>
 <widget class="QDialog" name="Avogadro::QtPlugins::CondaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>169</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Python Settings…</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="textLabel">
     <property name="text">
      <string>Only the “base” conda environment exists.
Would you like to create a new environment for Avogadro?
This will make a copy of your base environment.
</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Environment name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="environmentName">
       <property name="placeholderText">
        <string>avogadro</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Avogadro::QtPlugins::CondaDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Avogadro::QtPlugins::CondaDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
